### DISCUSSION TITLE
Query: target_cfgfile_load(): load file /home/gcb/Amiberry/conf/default.uae

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1190] [Q&A] [answered]
I'm just looking at revising the GUI configurations helptext, and stumbled upon this curio --- it looks to me like 'expected' behavior that has somehow gone undocumented? In any event, it ends up roping in GUI semantics at the same time.

The full verse & text missing, is when you start amiberry it searches for conf/default.uae, and if NOT found, merely uses the amiberry.conf file directives, and internal configurata. For example, if I start amiberry and **no** conf/default.uae exists, I _expect_ GUI -> Configurations panel to look like this;

![x1](https://imgur.com/BigOQZQ.png)

Conversely, if conf/default.uae exists, amiberry will load that file at start (which effectively redefines what the default startup configuration actually is), and I expect to see this, signifying the default.uae file has been loaded;

![x2](https://imgur.com/Apd3vMb.png)

Currently, if no conf/default.uae exists at start, the name/description fields are populated (as per image on wiki/GUI:-Configuration),
_inferring_ that config.uae file has been loaded...or such and similar ~ it doesn't visually say 'no config file found, using defaults instead' ...which is what the code does =) 

This can be confusing/misleading if the user has a conf/default.uae 'laying around' and they're unaware it will be loaded and trump the builtin defaults that would otherwise be used...ie; Quickstart panel ....if you're not aware you have a default.uae file with model=A500 & chipset=AGA at start amiberry->quickstart->do something A500 like OCS floppydisk file, YMMV =) 

With the name field already populated with the keyword 'default', imho increases the likelihood of that file existing. (in some minds that could be seen as an invitation of sorts, to setup the emulator how they want it, and then save this configuration to create the default.uae file that is 'apparently' missing ;)

What do you think? I'd be all for **not**  populating the name/description fields on the Configurations panel, in the case where no conf/default.uae could be found/used (and is not loaded). It more accurately represents the 'default' startup configuration wrt amiberry, as per image above.

As said though, I can't find any reference in docs pertaining to this ...feature... ; that is to say, if you always wanted amiberry to start with a particular/specific Amiga model/HW configuration, creating a default.uae file is a means to that end (and other stuff)... = feature ;) 

Easy enough to touch upon this point in helptext, but really the wiki/GUI:-Configuration page should note the case wrt if/ifnot exists default.uae, and how amiberry configures itself upon start when exists default.uae ?

TIA

**Comment by midwan** [comment, marked as answer]
Changed this behavior with d57accb7fc7cadfc767c17e8b5c638123f4102a1

**Reply by giantclambake** [reply]
Thanks ;)
